applications/CoSimulationApplication/python_scripts/solver_wrappers/kratos/kratos_base_wrapper.py

```diff
@@ -30,34 +48,49 @@ def __init__(self, settings, solver_name):
         # this creates the AnalysisStage, creates the MainModelParts and allocates the historial Variables on the MainModelParts:
         self._analysis_stage = self.__GetAnalysisStage()
 
+        if self.settings["solver_wrapper_settings"].Has("omp_num_threads"):
+            omp_num_threads = self.settings["solver_wrapper_settings"]["omp_num_threads"].GetInt()
```

Comment:
there is also a setting for specifying the number of threads in ProjectParameters.json no ?

Reply:
I mean of individual solvers (for kratos in general )?

Reply:
no there is nothing like that yet

From what I understand so far the user had to explicitly set `OMP_NUM_THREADS`

Reply:
ok .. Then perfect !

Reply:
By the way on a side note, having such a setting would we good. Just like here.